src/utils.ts

```diff
@@ -1,15 +1,44 @@
 import { version } from './version.js';
 
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+function detectRuntime() {
+  // Node.js
+  if (typeof process !== 'undefined' && process.versions?.node) {
+    return 'node';
+  }
+
+  // Cloudflare Workers
+  if (typeof navigator !== 'undefined' && navigator.userAgent === 'Cloudflare-Workers') {
+    return 'cloudflare-workers';
+  }
+
+  // Deno
+  // @ts-ignore
+  if (typeof Deno !== 'undefined') {
+    return 'deno';
+  }
+
+  // Browser
+  if (typeof window !== 'undefined') {
+    return 'browser';
+  }
```

Comment:
Do we want this? It kind of makes it look as if this SDK should be used in the browser.